python-sdk/example_dags/example_mssql_transform.py

```diff
@@ -56,24 +64,25 @@ def union_table_and_dataframe(input_table: Table, input_dataframe: pd.DataFrame)
     schedule_interval=None,
     start_date=START_DATE,
     catchup=False,
+    default_args=default_args,
 ) as dag:
     imdb_movies = aql.load_file(
         input_file=File(path="s3://astro-sdk/imdb_v2.csv"),
         task_id="load_csv",
-        output_table=Table(conn_id="mssql_conn"),
+        output_table=Table(conn_id=ASTRO_MYSQL_CONN_ID),
```

Comment:
Why did we replace the MS SQL connection with a MySQL one?

Reply:
@tatiana i see there were two conn one with MSSQL_CONN_ID and other MYSQL_CONN_ID, so I thought to use one

Reply:
They are different databases, as mentioned in https://github.com/astronomer/astro-sdk/pull/1937/files/3d38452bdab381179d900467a276824c87913c1f#r1211368378

Reply:
@tatiana i have updated MSSQL